lib/elixir/pages/Naming Conventions.md

```diff
@@ -0,0 +1,90 @@
+# Naming Conventions
+
+## Naming modules
+
+Elixir module names are written in "upper camel case", like `OptionParser`.
+
+Capital letters are kept in abbreviations, like `ExUnit.CaptureIO` or `Mix.SCM`.
```

Comment:
Ah. Yeah, seems to me that either making Mix smarter (or configurable?), or maybe just to improve its error messages or docs, might be preferable to documenting those specific details in these more general docs.